python_modules/dagster/dagster_tests/core_tests/test_object_store.py

```diff
@@ -28,6 +29,34 @@ def __init__(self):
         )
 
 
+class FancyStringFilesystemTypeStoragePlugin(TypeStoragePlugin):  # pylint:disable=no-init
+    @classmethod
+    def set_object(cls, object_store, obj, context, runtime_type, paths):
+        check.subclass_param(object_store, 'object_store', FileSystemObjectStore)
```

Comment:
I believe that subclass_param isn't what you want here and I'm surprised this works. This would only work is object_store is a Class rather than an Object. If it is a class we should name it object_store_cls or something